docs/installation-ios.md

```diff
@@ -159,3 +159,68 @@ didReceiveNotificationResponse:(UNNotificationResponse *)response
 If you're having problems with messages not being received, check out the following blog post for help:
 
 https://firebase.googleblog.com/2017/01/debugging-firebase-cloud-messaging-on.html
+
+
+
+## 5) Dynamic Links (optional)
+
+If you plan on using [Firebase Dynamic Links](https://firebase.google.com/docs/dynamic-links/) then, you need to:
+
+### 5.1)  create a new URL type
```

Comment:
I would consider adding screen shots for `5.1` and `5.2` as it is not noobs friendly (I know it is taken from firebase docs, but remember how we got lost in Xcode settings)